wordpress-mobile/AztecEditor-iOS
# Release/1.13.0

Updates the podspecs to release 1.13.0

To test:
 - Make sure the podspecs are correctly updated
 - Check the changelog is updated.
